src/game/generation/RegionGenerator.java

```diff
@@ -207,11 +237,11 @@ public void generateWorld(int x, int y) {
 		return blocks;
 	}
 
-	private int getAmplitude(BiomeType biometype) {
+	private static int getAmplitude(BiomeType biometype) {
 		int amplitude = 0;
 		switch (biometype) {
 		case OCEAN:
-			amplitude = 10;
+			amplitude = 3;
```

Comment:
Looks like a magic number

Reply:
Not really. Just want low amplitude for oceans.